spec/models/report_spec.rb

```diff
@@ -36,4 +36,29 @@
     user = create(:user, client_slug: "test")
     expect(Report.sql_for_user(user)).to include "WHERE client_slug='#{user.client_slug}'"
   end
+
+  it "#run", :elasticsearch do
```

Comment:
I think we've been including method names in the `describe` block in most specs and then using `it` to describe what the method does -- do you think we should move away from that here?